fly orgs show sometimes panics on non-existent slug

*Please only report specific issues with `flyctl` behavior*. Anything like a support request for your application should go to https://community.fly.io. More people watch that space and can help you faster!

**Describe the bug**
Briefly, describe what broke and provide the following details:

If you run `fly orgs show something-nonexistent`, it will sometimes output an error message looking like this:

```
$ fly orgs show something-nonexistent
Oops, something went wrong! Could you try that again?
```

while in other cases, a Go panic stacktrace is printed (see below).

* Operating system: macOS Monterey (12.6)
* `fly version`: `fly v0.0.414 darwin/amd64 Commit: 463a8297 BuildDate: 2022-10-17T21:15:34Z`

**Paste your `fly.toml`**

- No `fly.toml`

**Command output:**

```text
$ fly orgs show something-nonexistent
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: assignment to entry in nil map
[signal SIGSEGV: segmentation violation code=0xc000d9f528 addr=0x1ae96e0 pc=0x20211e7]

goroutine 1 [running]:
github.com/getsentry/sentry-go/internal/ratelimit.Map.Merge(0xc00098c900?, 0xc0004f2200?)
        github.com/getsentry/sentry-go@v0.12.0/internal/ratelimit/map.go:43 +0x108
github.com/getsentry/sentry-go.(*HTTPSyncTransport).SendEvent(0xc000986500, 0xc000472d80)
        github.com/getsentry/sentry-go@v0.12.0/transport.go:525 +0x4fc
github.com/getsentry/sentry-go.(*Client).processEvent(0xc0001971e0, 0xc000472d80, 0xc0005b0060, {0x2808dc0, 0xc0006321e0})
        github.com/getsentry/sentry-go@v0.12.0/client.go:553 +0x283
github.com/getsentry/sentry-go.(*Client).CaptureEvent(...)
        github.com/getsentry/sentry-go@v0.12.0/client.go:364
github.com/getsentry/sentry-go.(*Client).RecoverWithContext(0x60?, {0x0?, 0x0}, {0x22706c0?, 0x334a1b0?}, 0xc000d9f878?, {0x2808dc0, 0xc0006321e0})
        github.com/getsentry/sentry-go@v0.12.0/client.go:415 +0x235
github.com/getsentry/sentry-go.(*Client).Recover(0x0?, {0x22706c0?, 0x334a1b0?}, 0x0?, {0x2808dc0?, 0xc0006321e0?})
        github.com/getsentry/sentry-go@v0.12.0/client.go:379 +0x93
github.com/getsentry/sentry-go.(*Hub).Recover(0xc00095be60?, {0x22706c0?, 0x334a1b0?})
        github.com/getsentry/sentry-go@v0.12.0/hub.go:322 +0xc9
github.com/superfly/flyctl/internal/sentry.Recover({0x22706c0, 0x334a1b0})
        github.com/superfly/flyctl/internal/sentry/sentry.go:101 +0xae
main.run.func1()
        github.com/superfly/flyctl/main.go:38 +0x31
panic({0x22706c0, 0x334a1b0})
        runtime/panic.go:884 +0x212
github.com/superfly/flyctl/internal/command/orgs.runShow({0x281a8e0, 0xc000e18870})
        github.com/superfly/flyctl/internal/command/orgs/show.go:41 +0x87
github.com/superfly/flyctl/internal/command.newRunE.func1(0xc000563680, {0xc000e06440?, 0x1?, 0x1?})
        github.com/superfly/flyctl/internal/command/command.go:110 +0x17e
github.com/spf13/cobra.(*Command).execute(0xc000563680, {0xc000e06420, 0x1, 0x1})
        github.com/spf13/cobra@v1.2.1/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000dde000)
        github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bd
github.com/spf13/cobra.(*Command).ExecuteContextC(...)
        github.com/spf13/cobra@v1.2.1/command.go:911
github.com/superfly/flyctl/internal/cli.Run({0x281b558?, 0xc000cd0100?}, 0xc00053a000, {0xc00012c050, 0x3, 0x3})
        github.com/superfly/flyctl/internal/cli/cli.go:36 +0x21e
main.run()
        github.com/superfly/flyctl/main.go:45 +0x117
main.main()
        github.com/superfly/flyctl/main.go:26 +0x1e
```

--- a/internal/command/orgs/show.go
+++ b/internal/command/orgs/show.go
@@ -37,6 +37,9 @@ associated member. Details full list of members and roles.
 func runShow(ctx context.Context) (err error) {
 	client := client.FromContext(ctx).API()
 	selectedOrg, err := OrgFromFirstArgOrSelect(ctx)
+	if err != nil {
+		return err
+	}
 
 	org, err := client.GetDetailedOrganizationBySlug(ctx, selectedOrg.Slug)
 	if err != nil {

